tests/unit/Change.spec.ts

```diff
@@ -211,62 +213,63 @@ describe('Change component', () => {
     expect(wrapper.findComponent(YourCompany).exists()).toBe(false)
     expect(wrapper.findComponent(PeopleAndRoles).exists()).toBe(false)
 
-    store.state.stateModel.summaryMode = false
+    store.stateModel.summaryMode = false
     wrapper.vm.showFee = false
   })
 
   it('transactional folio number component renders for only premium accounts', async () => {
     expect(wrapper.findComponent(TransactionalFolioNumber).exists()).toBe(false)
-    store.state.stateModel.accountInformation.accountType = 'PREMIUM'
-    store.state.stateModel.summaryMode = true
+    store.stateModel.accountInformation.accountType = 'PREMIUM'
+    store.stateModel.summaryMode = true
     wrapper.vm.showFee = true
     // a wait needed as change to computed value triggers a re-rendering
     await Vue.nextTick()
 
     expect(wrapper.findComponent(TransactionalFolioNumber).exists()).toBe(true)
-    store.state.stateModel.accountInformation.accountType = ''
-    store.state.stateModel.summaryMode = false
+    store.stateModel.accountInformation.accountType = ''
+    store.stateModel.summaryMode = false
     wrapper.vm.showFee = false
   })
 
   it('Staff Payment, Court Order POA components display only for staff', async () => {
     expect(wrapper.findComponent(StaffPayment).exists()).toBe(false)
     expect(wrapper.findComponent(CourtOrderPoa).exists()).toBe(false)
-    store.state.stateModel.tombstone.keycloakRoles = ['staff']
-    store.state.stateModel.summaryMode = true
+    store.stateModel.tombstone.keycloakRoles = ['staff']
+    store.stateModel.summaryMode = true
     wrapper.vm.showFee = true
     // a wait needed as change to computed value triggers a re-rendering
     await Vue.nextTick()
 
     expect(wrapper.findComponent(StaffPayment).exists()).toBe(true)
     expect(wrapper.findComponent(CourtOrderPoa).exists()).toBe(true)
-    store.state.stateModel.tombstone.keycloakRoles = []
-    store.state.stateModel.summaryMode = false
+    store.stateModel.tombstone.keycloakRoles = []
+    store.stateModel.summaryMode = false
     wrapper.vm.showFee = true
   })
 
   it('resource subtitle states individual sole proprietors can change legal name', () => {
-    store.state.stateModel.tombstone.entityType = 'SP'
-    store.state.stateModel.peopleAndRoles.orgPeople = [
+    store.stateModel.tombstone.entityType = CorpTypeCd.SOLE_PROP
+    expect(CorpTypeCd.SOLE_PROP).toBe('SP')
```

Comment:
I think we probably don't need to test enums...?

Reply:
Sounds good, wasn't sure if you wanted to put it in there incase someone changed the `@\enums\` reference and thus cause an issue with all of the enums changing

Reply:
Removed enum checks

Reply:
If we were going to do this, then we should check all enums. But I'm not going to worry about this for now.